src/Faker/Provider/Base.php

```diff
@@ -62,6 +62,33 @@ public static function randomDigitNot($except)
         return $result;
     }
 
+    /**
+     * Generates a string of random digits
+     *
+     * @param int $length Length of the string to generate
+     * @return string A string of {@see $length} random digits
+     *
+     * @throws \OutOfBoundsException
+     * @throws \InvalidArgumentException
+     */
+    public static function randomDigitsString($length = 1)
```

Comment:
how is that different from `numerify('#######')`?

Reply:
I wasn't aware this function existed actually, which is why I made my own. It appears to work similarly and happens to work like intended so I guess I could remove my function. Should I commit my changes ?

Reply:
yes, please remove your function from the PR

Reply:
Done